R/rdeephaven/src/client.cpp

```diff
@@ -0,0 +1,113 @@
+#include <memory>
+#include <exception>
+#include <iostream>
+#include <sstream>
+#include <string>
+
+#include "deephaven/client/client.h"
+#include "deephaven/client/flight.h"
+
+#include <arrow/c/abi.h>
+#include <arrow/c/bridge.h>
+
+#include <Rcpp.h>
+
+
+// ######################### DH WRAPPERS, API #########################
+
+class TableHandleWrapper {
```

Comment:
If we can get this close to shipping, we can consider breaking this into a `.h` and a `.cpp`.  Right now, everything is defined in one place -- Java style.  Not a problem for now.